lms/djangoapps/courseware/grades.py

```diff
@@ -733,16 +619,10 @@ def get_score(user, block, scores_client, submissions_scores_cache, max_scores_c
     # older version of the problem -- they're still graded on what was possible
     # when they tried the problem, not what it's worth now.
     score = scores_client.get(block.location)
-    cached_max_score = max_scores_cache.get(block.location)
```

Comment:
I think this removal is causing the test errors - it needs to instead be replaced by the collected block_structure cache data.